hardware/2-0-revision.md

```diff
@@ -0,0 +1,272 @@
+| Pitch | 1.27mm, 80 way double sided.
+| Pads| 5 pads, with 4mm holes
+
+### Power supply
+
+Power to the micro:bit may be provided via 5V on the USB connector, or via a 3V battery plugged into the JST connector. It is also possible (with care) to power the micro:bit from the 3V /GND rings on the edge connector. The 3V /GND rings at the bottom can be used to supply power to external circuits. The board uses an LDO specified up to 300mA, with thermal cut-out for short circuit protection.
+
+| item          | details
+| ---           | ---
+| Operating range | 1.8V .. 3.6V
+| Operating current (USB and battery)   | 300mA max
+| On-board Peripherals budget | 90mA
+| Battery connector | JST S2B-PH-SM4-TB
+| Max current provided via edge connector | 190mA
+| More Info | [Power supply](./powersupply)
+
+### Interface
+
+The interface chip handles the USB connection, and is used for flashing new code to the micro:bit, sending and receiving serial data back and forth to your main computer.
+
+| item          | details
+| ---           | ---
+| Model         |[MKL27Z256VFM4](https://www.nxp.com/part/MKL27Z256VFM4#/)
+| Core variant: | [Arm Cortex-M0+](https://www.arm.com/products/processors/cortex-m/cortex-m0plus.php)
+| Flash ROM     | 256KB (128kB reserved for future use as storage)
+| RAM           | 16KB
+| Speed         | 48MHz
+| Debug capabilities | SWD
+| More Info | [DAPLink](/software/daplink-interface/), [KL27 reference manual (behind login)](https://www.nxp.com/webapp/Download?colCode=KL27P64M48SF6RM) [KL27 datasheet](https://www.nxp.com/docs/en/data-sheet/KL27P64M48SF6.pdf)
+
+### USB communications
+
+The micro:bit has an on-board USB communications stack, that is built into the firmware of the interface chip. This stack provides the ability to drag and drop files onto the
+MICROBIT drive in order to load code into the application processor. It also allows serial data to be streamed to and from the micro:bit application processor over USB to an external host computer, and supports the CMSIS-DAP protocol for host debugging of application programs.
+
+| item          | details
+| ---           | ---
+| Connector     | USB micro, MCR-B-S-RA-SMT-CS5-TR
+| USB version   | 2.0 Full Speed device
+| Speed         | 12Mbit/sec
+| USB classes supported | [Mass Storage Class (MSC)](https://en.wikipedia.org/wiki/USB_mass_storage_device_class)
+|    | [Communications Device Class (CDC)](https://en.wikipedia.org/wiki/USB_communications_device_class)
+| More Info | [DAPLink](/software/daplink-interface/)
+
+### Debugging
+
+The interface processor can be used with special host tools to debug code that is running on the application processor. It connects to the application processor via 4 signal wires. The KL27 interface processor code can also be debugged via it's integral SWD software debug interface, for example to load initial bootloader code into this processor at manufacturing time, or to recover a lost bootloader.
```

Comment:
I'm adding a comment in this file for V2.0 because I cannot add a comment to this line in the V2.2 doc, but the V2.2 version needs to remove the KL27 reference.